src/controller/python/test/unit_tests/test_generated_clusterobjects.py

```diff
@@ -0,0 +1,100 @@
+import unittest
+import chip.clusters as Clusters
+from rich.pretty import pprint
+from chip.clusters.Types import Nullable, NullValue
+
+'''
+This file contains tests for validating the generated cluster objects by running encoding and decoding
+through them symmetrically to exercise both pathways.
+'''
+
+
+class TestGeneratedClusterObjects(unittest.TestCase):
+    def CheckData(self, expected):
+        print("\nExpected Data:")
```

Comment:
Can we be quilter when the data is expected? -- A verbose output will make it hard to find the failed test.